### ISSUE TITLE
[Feature Request][NOWEB] - send group.join on participants join as well

### ISSUE THREAD

**Issue by johndoely** [allburov/whatsapp-http-api#583]
**Describe the bug**

> Webhook event group.join is not coming to the webhook URL for NOWEB engine. (as seen at this table https://waha.devlike.pro/docs/how-to/webhooks/#webhooks)
> The only case it works is when the waha connected user joins the group, but the main goal is to 'watch' other users joining the group. For WEBJS it is working normally. 

**Version**

```json
{
  "version": "2024.10.3",
  "engine": "NOWEB",
  "tier": "PLUS",
  "browser": "/usr/bin/chromium"
}
```

**To Reproduce** Steps to reproduce the behavior:

1. Set webhook url to your preferred url
2. Use own waha connected user to join a group, you can see the incoming webhook data
3. Join the group with an other whatsapp
4. No incoming webhook data

**Expected behavior**

> We should expect incoming data to webhook url when any other user joins the group.

**Engine** I'm using **NOWEB** engine.

**Docker Logs**

> It is possible only to see logs of own waha connected user, other users don't log error.

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
yeah, that webhook is supposed to fire when YOU join a new group, now when someone joins it.
To get participants you can use `GET /api/{session}/groups/{groupId}/participants`
https://waha.devlike.pro/docs/how-to/groups/#participants

Just do not request it too frequently, it does communicate with WA servers and you can get rate limited, like in https://github.com/devlikeapro/waha/issues/462

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Probably we can add `groups.participants.{action}` tho :thinking: Let me know if you're interested and we can see how many people are interested in that too!

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by johndoely** [comment]
We can see that in WEBJS it works... Here I have two images of the payload I received on my n8n containing group.join from two -different- numbers than the one monitoring... This feature would be great in NOWEB since it is way more cost effective and less demanding than requesting group participants every now and then (and hoping not to get blocked).

<img width="50%" alt="Captura de Tela 2024-10-11 às 19 21 58" src="https://github.com/user-attachments/assets/51beff9d-0610-4cfb-a224-b12bef0bac82"><img width="50%" alt="Captura de Tela 2024-10-11 às 19 18 19" src="https://github.com/user-attachments/assets/bff3df90-0461-47a8-8512-a8ea9567270d">


[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
I see, ok, let me see what we can do here!

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by melquisedeque-magalhaes** [comment]
+1

**Comment by marinaglancy** [comment]
+1

I have to listen to the **engine.event** event with `payload.event='group-participants.update'`

I would like not to listen to the engine.event because it is way too noisy.

**Comment by manojsitapara** [comment]
+1

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Added in `2025.2.4`
- Add [👥 group.v2 events](https://waha.devlike.pro/docs/how-to/groups/#events)
  - group.v2.join - when you join a group
  - group.v2.leave - when you leave a group
  - group.v2.participants - when someone joins, leaves, get promoted or demoted in a group
  - group.v2.update - when group information are updated
  - WEBJS, NOWEB, GOWS engines

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
